upup/pkg/fi/cloudup/awstasks/subnet.go

```diff
@@ -246,6 +248,10 @@ func (_ *Subnet) RenderTerraform(t *terraform.TerraformTarget, a, e, changes *Su
 		AvailabilityZone: e.AvailabilityZone,
 		Tags:             e.Tags,
 	}
+	netnum := fi.IntValue(e.AwsIpv6SubnetNum)
+	if netnum > 0 {
```

Comment:
Isn't the all-zero /64 subnet valid? Shouldn't this be testing for non-`nil` instead?